assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -1393,7 +1393,18 @@ const baseSelectors = {
 			return undefined;
 		}
 
+		const isGA4DashboardView =
+			select( MODULES_ANALYTICS ).isGA4DashboardView();
+
 		return Object.keys( modules ).reduce( ( acc, slug ) => {
+			if ( slug === 'analytics' && isGA4DashboardView ) {
+				return acc;
+			}
+
+			if ( slug === 'analytics-4' && ! isGA4DashboardView ) {
+				return acc;
+			}
```

Comment:
Similarly to my other comment, I think these would be better off with the conditions combined:

```suggestion
			if (
				( slug === 'analytics' && isGA4DashboardView ) ||
				( slug === 'analytics-4' && ! isGA4DashboardView )
			) {
				return acc;
			}
```

Reply:
Updated ✅